### ISSUE TITLE
Crop unnecessary parts of T1w images

### ISSUE THREAD

**Issue by PaulBautin** [sct-pipeline/csa-atrophy#44]
"spine-generic" T1w images have an unnecessarily large "Field Of View" that slows down the pipeline. Could we crop images with `sct_crop_image`. @jcohenadad  is there a procedure for this?

**Comment by jcohenadad** [comment]
you could do something like this:
~~~
# Detect ponto-medullary junction
sct_detect_pmj -i /Users/julien/data/spine_generic_single_20200708/results/data/sub-douglas/anat/sub-douglas_T1w.nii.gz -c t1 -qc qc
# check for QC, if problem, do it manually, push to the dataset, and run a 2nd pass
# run a small python script that will fetch the `z_pmj` coordinate of the label
# note: you can also run: sct_label_utils -i IMAGE_pmj.nii.gz -display
# fetch n_x dim
# Crop image
sct_crop_image -i IMAGE -xmin nx/2-20 -xmax nx/2+20 -zmax z_pmj 
~~~

This gives you something like this:
![image](https://user-images.githubusercontent.com/2482071/90444176-6a8ee300-e0ab-11ea-92cd-316251e49f63.png)

**Comment by PaulBautin** [comment]
Issue was fixed in PR #42. Closing issue.
